Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -319,3 +336,11 @@ private func profileController() -> UIViewController {
 
   return BackerDashboardViewController.instantiate()
 }
+
+private func extractViewController(_ viewController: UIViewController) -> UIViewController? {
+  if let navigationController = viewController as? UINavigationController {
+    return navigationController.viewControllers.count == 1 ? navigationController.viewControllers.first : nil
```

Comment:
Only extract embedded view controller if we're popped all the way to the top of the stack..we don't want to scroll view controllers on the stack if they are overlapped with other controllers.